cms/templates/cms/home_page.html

```diff
@@ -9,54 +9,115 @@
-      </div>
+<div class="main-content-wrapper app-media">
+  <section class="main-content">
 
-    </div>
-    <div class="login-button-wrapper">
-      {% if authenticated %}
-      <h1>Welcome, {{ username }}</h1>
-      {% else %}
-      <a class="login-button button" href="{% url 'social:begin' 'edxorg' %}">Log in with edx.org</a>
-      {% endif %}
-    </div>
-    <div class="programs-container">
-        {% for program in programs %}
-          <div class="program-box">
-            <span style="line-height: 115px"></span>
-            {% if program.programpage %}
-            <a href="{{ program.programpage.url }}">
-              <div class="program-name">{{ program }}</div>
+
+    <div class="banner-wrapper">
+      <div class="banner-wrapper-content">
+        <h1>
+          {{ title }}
+        </h1>
+        <a href="add" role="button" class="btn btn-primary btn-lg btn-learn-more">
+          <span>LEARN MORE</span>
+        </a>
+      </div>
+      <div class="banner-bottom">
+        <div class="banner-wrapper-content">
+          <h2>Get Started Today!
+            <a href="{% url 'social:begin' 'edxorg' %}" role="button" class="btn btn-success sign-in-button margin-left-10">
+              <span>Sign in with edX.org</span>
             </a>
-            {% else %}
-              {{ program }}
-          {% endif %}
-          </div>
-        {% endfor %}
+          </h2>
+        </div>
+      </div>
     </div>
 
-    <div class="footer-row">
-        <div class="footer-links">
-            <a href="http://giving.mit.edu/">MIT Giving</a>
-            <a href="https://www.edx.org/">edX</a>
-            <a href="https://odl.mit.edu/">Office of Digital Learning</a>
+
+    <div class="panel panel-main-content">
+      <div class="panel-body">
+        <h3>Micromasters</h3>
+        <div class="media-list is-grid padding-bottom-50">
+          <ul class="blocks blocks-100 blocks-xlg-4 blocks-lg-3 blocks-md-3 blocks-ms-2 blocks-xs-2" data-plugin="animateList" data-child=">li">
+
+            {% for program in programs %}
+            <li>
+              <div class="media-item">
+                <div class="media-top">
+                  <div class="image-wrap">
+                    <img class="image" src="static/images/course-1.png" alt="..."> 
+                  </div>
+                </div>
+                <div class="info-wrap">
+                  {% if program.programpage %}
+                  <a href="{{ program.programpage.url }}">
+                    <div class="title">{{ program }}</div>
+                  </a>
+                  {% else %}
+                  <div class="title">{{ program }}</div>
+                  {% endif %}
+                  <div class="description">
+                    {{ program.description }}
+                  </div>
+                  <div class="media-footer">
+                    <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} courses</span></div>
```

Comment:
edge case here:
you need to take in account that there might me a single course and in this case `courses` should be `course`